src/components/ConfirmationDialog.js

```diff
@@ -17,8 +17,7 @@ const ConfirmationDialog = ({
       sx={{
         display: 'flex',
         flexDirection: 'column',
-        height: '120px',
-        zIndex: 2
+        height: '140px'
```

Comment:
add prop height and leave the default value 120px

Reply:
![Uploading image.png…]()